tests/hikari/test_embeds.py

```diff
@@ -62,3 +62,22 @@ def test_proxy_filename(self, resource_with_proxy):
     def test_proxy_filename_when_resource_is_none(self, resource_with_proxy):
         resource_with_proxy.proxy_resource = None
         assert resource_with_proxy.proxy_filename is None
+
+
+class TestEmbedTotalLength:
+    @pytest.fixture()
+    def embed(self):
+        embed = embeds.Embed()
+        return embed
+
+    def test_total_length_when_embed_is_empty(self, embed):
+        assert embed.total_length() == 0
+
+    def test_total_length(self, embed):
```

Comment:
For better test coverage, there should also be cases for when each of them are None or not.

You can check the html files generated by `nox -s pytest` under `public/` for coverage statistics :)

Reply:
Ah yea I was going to do that initially but thought it was pedantic. Will add now

Reply:
Added functions, full coverage now